README.md

```diff
@@ -1,6 +1,6 @@
 # Scalyr data source for Grafana
 
-The Scalyr Grafana data source plugin allows you to create and visualize graphs and dashboards in Grafana using data in Scalyr. You may want to this to visualize Scalyr data next to other data sources when you want to monitor many feeds on a single dashboard. 
+The Scalyr Grafana data source plugin allows you to create and visualize graphs and dashboards in Grafana using data in Scalyr. You may want to use this plugin to allow yo to to visualize Scalyr data next to other data sources when you want to monitor many feeds on a single dashboard. 
```

Comment:
I think there is still a typo "allow yo to" should be "allow you" don't need the extra "to".